sections/summary.tex

```diff
@@ -0,0 +1,105 @@
+This field is fully observed within the dataset, and each article only has a single author, so we do not account for multiple authors.
+There are $227$ different authors within the dataset, but after preprocessing $184$ remain.
+
+The category field describes a variety of different aspects, but the categories are generally either about a specific newspaper or an overall subject.
+This field is also fully observed within the dataset, and there are $58$ unique categories in the dataset before processing, and $34$ categories after preprocessing.
+
+The taxonomy field describes a hierarchical sequence of the topical or geographical subject of the articles.
+Each sequence consists of a number of sub-taxonomies.
+This field is only partially observed within the dataset, with ${\sim}25\%$ of the articles containing this field.
+There are $1135$ different taxonomies, but after preprocessing $345$ remain.
+
+%models
+The models we have made that include metadata for our evaluations are called the author-topic, category-topic, and taxonomy-topic model.
+We also use a standard \gls{lda} model as a baseline for the performance of our models.
+In \gls{lda}, $D$ is the number of documents in the corpus, $N_d$ is the number of words in document $d$, and $K$ is the number of topics.
+Topics are represented as distributions over words and documents are represented as distributions of topics.
+
+In the author-topic and category-topic models, there are no document-topic distributions $\theta$.
+Instead, each of the $A$ authors and $C$ categories has its own topic distribution.
+This is based on the assumption that authors prefer to write about specific topics, and that categories of the articles were chosen based on the content of the finished article or that local newspapers have their own unique topic preferences.
+For our category-topic model and author-topic model, each document $d$ is associated with one category $c_d$ from a set of categories $C$ and one author $a_d$ from a set of authors $A$, which is used when drawing a word topic.
+
+To have our taxonomy-topic model handle the hierarchical structure of the taxonomy metadata field, we use a hierarchical topic model called the \acrfull{pam}.
+Pachinko allocation generalizes \gls{lda}, making it possible to construct topic hierarchies based on any \gls{dag} structure.
+\gls{pam} is a topic model focusing on finding topics of different abstraction levels and modeling the connections between these topics.
+Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the topics will not actually be based on the taxonomy.
+To account for this, we lock taxonomy for the words in the observed documents to be in the corresponding topics within the pachinko \gls{dag} instead of continuously sampling them using Gibbs sampling.
+This creates a constant context for the taxonomy topics, which the documents with unobserved taxonomies will be fitted around.
+
```

Comment:
Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the taxonomy values would be disregarded during training since it would generate new taxonomy sequences.
However, in our case, we have a dataset with a partially observed taxonomy field (${\sim}25\%$ of the documents), and we want to use the existing metadata information to estimate the topics quicker and more accurately.
To account for this, we only sample the unobserved nodes within the topic sequences.
For some of the dataset only the fourth layer is unobserved, but for the ${\sim}75\%$ documents without a taxonomy sequence, all layers are unobserved.
The observed taxonomy sequences are never sampled, hence they are 'locked' in place.
This creates a constant context for the taxonomy topics, which the documents with unobserved taxonomy sequences will be fitted around.